assets/maps/mining_magnet/syndicate.dmm

```diff
@@ -0,0 +1,511 @@
+/obj/item/currency/spacecash/thousand,
+/obj/item/currency/spacecash/thousand,
+/obj/item/currency/spacecash/thousand,
+/obj/item/currency/spacecash/thousand,
+/obj/item/currency/spacecash/thousand,
+/obj/item/currency/spacecash/thousand,
+/obj/item/currency/spacecash/thousand,
+/obj/item/currency/spacecash/thousand,
+/obj/item/storage/secure/ssafe/syndicate_mining_encounter{
+	pixel_y = 25
+	},
+/turf/unsimulated/floor/shuttle/red{
+	dir = 4
+	},
+/area/mining/magnet)
+"y" = (
+/obj/stool/bed,
+/obj/item/clothing/suit/bedsheet{
+	bcolor = "red";
+	icon_state = "bedsheet-red";
+	name = "syndicate bedsheet"
+	},
+/mob/living/critter/human/syndicate/rifle,
+/turf/unsimulated/floor/shuttle/red{
+	dir = 4
+	},
+/area/mining/magnet)
+"A" = (
+/obj/machinery/door/airlock/pyro/glass{
+	dir = 4
+	},
+/turf/unsimulated/floor/shuttle/red{
+	dir = 4
+	},
+/area/mining/magnet)
+"D" = (
+/obj/table/syndicate/auto,
+/turf/unsimulated/floor/shuttle/red{
+	dir = 4
+	},
+/area/mining/magnet)
+"E" = (
+/turf/unsimulated/floor/shuttle/red{
+	dir = 4
+	},
+/area/mining/magnet)
+"F" = (
+/obj/machinery/door/airlock/pyro/glass{
+	dir = 1
+	},
+/turf/unsimulated/floor/shuttle/red{
+	dir = 4
+	},
+/area/mining/magnet)
+"H" = (
+/obj/storage/crate/classcrate{
+	name = "money crate"
+	},
+/obj/item/currency/spacecash/thousand,
+/obj/item/currency/spacecash/thousand,
+/obj/item/currency/spacecash/thousand,
+/obj/item/currency/spacecash/thousand,
+/obj/item/currency/spacecash/thousand,
+/obj/item/currency/spacecash/thousand,
+/obj/item/currency/spacecash/thousand,
+/obj/item/currency/spacecash/thousand,
+/obj/item/currency/spacecash/thousand,
+/obj/item/currency/spacecash/thousand,
+/turf/unsimulated/floor/shuttle/red{
+	dir = 4
+	},
+/area/mining/magnet)
+"I" = (
+/obj/storage/closet/syndicate,
+/obj/item/clothing/shoes/swat,
+/obj/item/clothing/shoes/swat,
+/obj/item/clothing/gloves/black,
+/obj/item/clothing/gloves/black,
+/obj/item/clothing/head/bandana/random_color,
+/obj/item/clothing/head/bandana/random_color,
+/obj/item/clothing/head/black,
+/obj/item/clothing/head/black,
+/turf/unsimulated/floor/shuttle/red{
+	dir = 4
+	},
+/area/mining/magnet)
+"J" = (
+/obj/stool/bed,
+/obj/item/clothing/suit/bedsheet{
+	bcolor = "red";
+	icon_state = "bedsheet-red";
+	name = "syndicate bedsheet"
+	},
+/obj/machinery/light/small,
+/turf/unsimulated/floor/shuttle/red{
+	dir = 4
+	},
+/area/mining/magnet)
+"K" = (
+/turf/simulated/floor/auto/glassblock/red,
+/area/mining/magnet)
+"M" = (
+/obj/stool/bed,
+/obj/item/clothing/suit/bedsheet{
+	bcolor = "red";
+	icon_state = "bedsheet-red";
+	name = "syndicate bedsheet"
+	},
+/obj/item/storage/secure/ssafe/syndicate_mining_encounter{
+	pixel_y = 5;
+	pixel_x = -24
+	},
+/turf/unsimulated/floor/shuttle/red{
+	dir = 4
+	},
+/area/mining/magnet)
+"N" = (
+/mob/living/critter/human/syndicate/knife,
+/turf/unsimulated/floor/shuttle/red{
+	dir = 4
+	},
+/area/mining/magnet)
+"O" = (
+/mob/living/critter/human/syndicate/rifle,
```

Comment:
These syndicate NPCs are kind of weird and not very strong, I solo'd the entire thing with an uncharged pickaxe and only took ~40 damage. Consider replacing them with a crew of gunbots?

Reply:
What were you wearing? The assault rifle syndicates do a ton of damage I thought